platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/logging/SimpleTestEventLogger.java

```diff
@@ -66,12 +69,27 @@ public void completed(TestDescriptorInternal descriptor, TestResult result, Test
 
                 // Print the failure message(s)
                 for (TestFailure failure : result.getFailures()) {
-                    if (failure.getDetails().getMessage() != null) {
-                        output.append("    ").withStyle(StyledTextOutput.Style.Identifier).append(failure.getDetails().getClassName());
-                        output.append(": ").println(failure.getDetails().getMessage());
+                    TestFailureDetails details = failure.getDetails();
+                    if (!TextUtil.isBlank(details.getMessage())) {
```

Comment:
💭 If no message provided, should this print some default "<No failure info>" message?

Reply:
I updated the test to show what this looks like.

We show the path to the test and say it failed, but we don't have any other info.